### DISCUSSION TITLE
How to use show dimensions in react?

### DISCUSSION THREAD

**Discussion by AtakishiAliyev** [google/model-viewer#4235] [Q&A] [answered]
Hi, I'm using the model viewer with React. I want to implement [Show Dimensions](https://modelviewer.dev/examples/annotations/#dimensions) in the model viewer's own documentation, but it doesn't work in React. How can I solve this problem?

For example: getBoundingBoxCenter not working with react

**Comment by elalish** [comment, marked as answer]
I don't see any reason one API would fail in React while the others work. What version of MV are you using? Can you show a repro link where `getBoundingBoxCenter` isn't working, or share an error message or something?

**Reply by AtakishiAliyev** [reply]
> I don't see any reason one API would fail in React while the others work. What version of MV are you using? Can you show a repro link where `getBoundingBoxCenter` isn't working, or share an error message or something?

How can we run the show dimensions example given in the docs in react? Is there a sample code about it?

**Reply by elalish** [reply]
I don't have any experience with React, but others are using MV there. Search our discussions and you should find some useful threads.

**Comment by AtakishiAliyev** [comment]
thanks
